docs/UserGuide.md

```diff
@@ -1,156 +1,224 @@
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find 'hiking'`
+* `find 'CS2103T' 'group meeting' 'project'`
 
-### 2.6. Deleting a person : `delete`
+### Deleting tasks : `delete`
 
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Deletes the specified tasks from to-do list. It is irreversible.<br>
+You can delete more than one task
+Format: `delete INDEX [INDEX...]`
 
-> Deletes the person at the specified `INDEX`. <br>
+> Deletes the tasks at the specified `INDEX`.<br>
 > The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+> The index must be a **positive integer** 1, 2, 3, ...
 
 Examples:
 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
+* `list finished`<br>
+  `delete 1 2`<br>
+  Deletes the 1st and the 2nd tasks in the result of `list` command.
+* `find 'party'`<br>
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the 1st task in the results of the `find` command.
 
-### 2.7. Select a person : `select`
+### Marking tasks as finished : `finish`
 
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+Mark the specified tasks as finished.<br>
+Format: `finish INDEX [INDEX...]`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
+> Mark the tasks at the specified `INDEX` as finished.<br>
 > The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+> The index must be a **positive integer** 1, 2, 3, ...<br>
+> If the tasks at the specified `INDEX` are already finished, no change made.
 
 Examples:
 
+* `list pending \in this week`<br>
+  `finish 1 2`<br>
+  Marks the 1st and the 2nd tasks in the result of `list` command as finished.
+* `find 'project'`<br>
+  `finish 1`<br>
+  Mark the 1st task in the results of the `find` command as finished.
+
+### Editing an existing task : `edit`
+
+Edit the specified task.<br>
+Format 1: `edit INDEX [\desc TASK_DESCRIPTION] [\from START_TIME] [\to END_TIME] [\remind_at REMIND_TIME] [\every RECURRENCE_INTERVAL] [\as PRIORITY] [\under TAG...]`
+Format 2: `edit INDEX [\desc TASK_DESCRIPTION] [\by TIME] [\remind_at REMIND_TIME] [\every RECURRENCE_INTERVAL] [\as PRIORITY] [\under TAG...]`
+
+> The rule for `edit` command is the same as `add` command
+
+Examples:
 * `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br>
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  `edit 1 \desc watch NBA \remind_at this Sunday 8am`<br>
+  Suppose originally, the description of this task is `watch nba`,
+  it will be change to `watch NBA`.
+  The reminder time will also be updated to the specified one: `this Sunday 8am`.
+
```

Comment:
As your parameters are optional, more thinking can be placed for this `edit` function.
Maybe you can explain more, like what will happen to the existing values that are not specified in a particular `edit` command.
Eg. edit 2 \desc Assignment
- What will happen to the dates that were earlier on but not included now?
- Or what if I intend to leave out a start time to convert it from a task that only have deadline, etc.